apps/studio/src/components/plugins/PluginList.vue

```diff
@@ -10,7 +10,7 @@
       <div class="info">
         <div class="title">
           {{ plugin.name }}
-          <span class="badge" v-if="$bksConfig.plugins?.[plugin.id]?.enabled === false">disabled</span>
+          <span class="badge" v-if="$bksConfig.plugins?.[plugin.id]?.disabled">disabled</span>
```

Comment:
could one also use `bksConfig.get`? Or is that API not exposed?

Reply:
Actually that's better, idk why I was not very convinced to use `bksConfig.get` on this one.